WcaOnRails/app/webpacker/components/Delegates/index.jsx

```diff
@@ -35,12 +36,28 @@ export default function Delegates() {
     [delegateGroups],
   );
 
-  const [activeRegion, setActiveRegion] = React.useState();
-  const [adminMode, setAdminMode] = React.useState(false);
+  const [hash, setHash] = useHash();
+
+  const SelectedComponent = React.useMemo(() => {
+    const selectedRegionIndex = delegateRegions.findIndex((region) => region.id === +hash);
+    if (selectedRegionIndex === -1) {
+      setHash(delegateRegions[0]?.id);
+      return () => null;
+    }
+    const selectedSection = delegateRegions[selectedRegionIndex];
+    if (selectedSection.component) {
```

Comment:
selectedSection is an item of delegateRegions and delegateRegions is fetched from server. But delegateRegions doesn’t have component key. Correct me if I’m wrong.